aiohttp/test_utils.py

```diff
@@ -466,18 +459,21 @@ def setup_test_loop(loop_factory=asyncio.new_event_loop):
     return loop
 
 
-def teardown_test_loop(loop):
+def teardown_test_loop(loop, fast=None):
     """Teardown and cleanup an event_loop created
     by setup_test_loop.
 
     """
+    if fast is None:
```

Comment:
I think this should be

```python
fast | = TESTS_FAST
```

that way if you're testing with pytest and have `AIOHTTP_TESTS_FAST=True` but don't supply the `--fast` argument you get fast tests.

With current setup `AIOHTTP_TESTS_FAST` has no effect with pytest.